docs/page-controller.md

```diff
@@ -130,8 +132,55 @@ function add_report_menu_item( $report_pages ) {
 add_filter( 'woocommerce_admin_report_menu_items', 'add_report_menu_item' );
 ```
 
+#### Example - Adding a New WooCommerce Admin Page
+
+Alternatively, register a regular page with the controller.
+
+```php
+function add_extension_register_page() {
+    if ( ! function_exists( 'wc_admin_register_page' ) {
```

Comment:
One final syntax error - missing a closing `)`.

Reply:
@psealock looks like this was merged into `main` with the syntax error.. can you fix please?

Reply:
Thank you! https://github.com/woocommerce/woocommerce-admin/pull/5876